stable/stolon/templates/keeper-podmonitor.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.monitoringPG.enabled }}
```

Comment:
We already have `serviceMonitor` for other metrics, so it makes sense to create serviceMonitor for these metrics for consistency.
Also, `monitoring.coreos.com` is not the only method for scraping, so you should have:
* add metrics to the `keeper-headless-service.yaml` 
*  check both flags: serviceMonitor.enabled and monitoringPG.enabled.